pkg/controllers/cassandra/nodepool/resource.go

```diff
@@ -62,6 +63,29 @@ func StatefulSetForCluster(
 					},
 					InitContainers: []apiv1.Container{
 						pilotInstallationContainer(&cluster.Spec.NavigatorClusterConfig.PilotImage),
+						apiv1.Container{
+							Name:            "install-agent-bond",
+							Image:           "fabric8/java-alpine-openjdk8-jre:latest",
```

Comment:
Instead of using an image that contains a full jre just to grab a jar file out of it, can we include this one jar in the pilot-cassandra image and adjust this initContainer to use this? By both initContainers referencing the same image, it saves it being pulled multiple times 😄